docs/apis/calcite.md

```diff
@@ -0,0 +1,235 @@
+Apache Apex, unified stream processing engine enables application developers to process data at very high throughput with low latency. Although the different types of data have different processing needs, the SQL remains a popular and a generic way for processing data. To ensure that existing ETL developers and developers who are well versed with Database applications adopt stream processing application development with ease, integration of SQL with Apex was needed. Being popular Apache project, Apache Calcite was chosen for the integration and to facilitate the use of SQL with Apex.
```

Comment:
"Apache Apex, unified stream processing engine enables" => Apache Apex is a unified stream and batch processing engine that enables
"the SQL" => SQL
"Being popular" => Being a popular
"was chosen ..." => was chosen for this purpose and its integration with Apex is described below.